systemtest/src/main/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -817,6 +817,32 @@ public void waitForClusterAvailabilityTls(String userName, String namespace) thr
         }
     }
 
+    /**
+     * Wait for cluster availability with specific cluster name, check availability of external routes with TLS
+     * @param userName user name
+     * @param namespace cluster namespace
+     * @param clusterName cluster name
+     * @throws Exception exception
+     */
+    public void waitForClusterAvailabilityTls(String userName, String namespace, String clusterName) throws Exception {
```

Comment:
Instead of duplicate whole method, you cat just add new parameter and then override it without `clusterName` parameter and put there default value `CLUSTER_NAME`

Reply:
yes sure.. ) good point